service/src/main/java/greencity/service/ubs/SuperAdminServiceImpl.java

```diff
@@ -96,6 +96,13 @@ public class SuperAdminServiceImpl implements SuperAdminService {
     private static final String RECEIVING_STATIONS_OR_COURIER_EXIST_MESSAGE =
         "Current receiving stations: %s or courier: %s don't exist.";
     private static final String REGION_OR_COURIER_EXIST_MESSAGE = "Current region: %s or courier: %s don't exist.";
+    private static final String REGION_OR_RECEIVING_STATIONS_EXIST_MESSAGE =
```

Comment:
I see in the constant name `STATIONS_EXIST` and in the message that `stations: %s don't exist`

Reply:
Yes, I change it